girder/models/setting.py

```diff
@@ -215,6 +215,22 @@ def validateCoreUserDefaultFolders(self, doc):
                 'User default folders must be either "public_private" or '
                 '"none".', 'value')
 
+    def validateCoreRouteTable(self, doc):
```

Comment:
Note that this will break after #1483 ... we should rebase this on top of that change once it's merged.